.travis.yml

```diff
@@ -10,23 +10,20 @@ addons:
 cache:
   yarn: true
   directories:
-    - $HOME/.cache
+    - $HOME/.npm
 
 env:
   global:
     # See https://git.io/vdao3 for details.
     - JOBS=1
 
-before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash
-  - export PATH=$HOME/.yarn/bin:$PATH
 
 install:
-  - yarn
+  - npm install
 
 script:
-  - yarn run lint:js
-  - yarn test
+  - npm run lint:js
+  - npm test
 
 deploy:
```

Comment:
If you remove the deploy section (because it doesn't work) then we should be able to merge this and get the production website updated 👍